ddistro_install.sh

```
@@ -0,0 +1,42 @@
+
+echo 
+echo "This will start CHIM XTTS to download the selected model"
+echo "Wait for the message 'Uvicorn running on http://0.0.0.0:8020 (Press CTRL+C to quit)'"
+echo "Then close this window. Press ENTER to continue"
+read
+
+echo "please wait...."
+
+python -m xtts_api_server --listen
+
+echo "Press Enter
```

Shellcheck (0.10.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)